### DISCUSSION TITLE
sudoku graph

### DISCUSSION THREAD

**Discussion by mjhajharia** [networkx/networkx#5085] [Ideas]
hi! i checked the sudoku graph generator in networkx, and if you people think it's a good idea we can maybe have a version of this as a notebook or maybe a figure in the documentation. might encourage more people to view graph theory as fun!

https://gist.github.com/mjhajharia/bed5e13ab84660456323648d6d99d2fd

**Comment by dschult** [comment]
Yes! A notebook or similar would fit well in the networkx/nx-guides repository.
A relatively short example could fit into the "examples" gallery of the documentation (networkx/networkx repo in the examples directory). You can put together a PR in either location and we'll work to get it in. let us know (?repost here?) if you have questions about that.

**Reply by rossbar** [reply]
Agreed! I took a quick look at the linked gist and got the impression that this would make a nice gallery example. I'd recommend to start there and then potentially add a notebook to `nx-guides` to expand on the gallery example if there were desire to do so. Of course, it's up to you and we're happy to help you get started either way!

**Reply by mjhajharia** [reply]
hi! i improved it and formatted it for nx-guides, if this looks okay i'll make a PR https://nbviewer.jupyter.org/gist/mjhajharia/1a53f3cfdfe7c0ddfb4d47b5a7a3e5e5
